docs/examples.rst

```diff
@@ -0,0 +1,310 @@
+Examples
+========
+
+Creating a yogadl.Storage
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Most users will interact with :class:`yogadl.Storage` object as a mechanism for
+storing and fetching datasets.  The simplest ``Storage`` is the
+:class:`yogadl.storage.LFSStorage`, or "local filesystem storage".  Let's create
```

Comment:
maybe: call out our other storage classes or link to API

Reply:
Good idea